File: pset6/sentimental/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 0 and height < 24:
        break
    else:
        print("Height must be a positive integer less than 24.")

width = height + 1
spaces = width - 2
hashes = width - spaces

for i in range(height):
    print(" " * spaces, end="")
    spaces -= 1
    print("#" * hashes, end="")
    hashes += 1
    print()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>
// This is sort of a combination of the less comfortable and more comfortable solutions
int main(void)
{
    // Get user input for height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 0 | height > 23); // Must be between 0 and 23

    for (int i = 0; i < height; i++) // Repeats inside loop or determines height, simply put
    {
        for (int j = 0; j < height-i-1; j++) // Makes number of spaces per row, decrementing by 1 each time.
        {
            printf(" ");
        }
        for (int k = 0; k < i+2; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 0; k < i+2; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}
